components/button.css

```diff
@@ -0,0 +1,83 @@
+.button-allign {
+    margin-left: 1rem;
+    margin-bottom: 1rem;
+}
+
+.buttons {
+    font-size: 1rem;
+    padding: 0.5rem;
+    border-radius: 5px;
+}
```

Comment:
Put a `cursor: pointer` here also to make understand the user something interactive.